Sources/Tabs/ProfileCoordinator.swift

```diff
@@ -36,4 +40,26 @@ final class ProfileCoordinator: NSObject,
     func updateToken(_ tokenHolder: TokenHolder) {
         baseVc?.updateToken(tokenHolder)
     }
+
+    func openSignOutPage() {
+        let navController = UINavigationController()
+        let vm = SignOutPageViewModel(analyticsService: analyticsCenter.analyticsService) { [unowned self] in
+          do {
+            analyticsCenter.analyticsPreferenceStore.hasAcceptedAnalytics = false
+            analyticsCenter.analyticsService.denyAnalyticsPermission()
+            // TODO: DCMAW-8933 will handle sign out error scenarios
+            try? userStore.clearTokenInfo()
+            try? userStore.secureStoreService.delete()
+            userStore.shouldPromptForAnalytics = true
+              root.dismiss(animated: false) {
+                  self.finish()
+              }
+          } catch {
+            print(error.localizedDescription)
+          }
+        }
+        let signoutPageVC = GDSInstructionsViewController(viewModel: vm)
+        navController.setViewControllers([signoutPageVC], animated: true)
+        root.present(navController, animated: true)
+      }
```

Comment:
I am wondering slightly if this would be more future-proof to be made into a coordinator opened modally but happy to continue with this for the meantime 👍

Reply:
Do you mean make something like a SignoutCoordinator which then works in a similar way to how the MainCoordinator opens the LoginCoordinator?

Reply:
Yes, exactly 🙂
Something to think about for the future, definitely not necessary at the moment!